README.md

```diff
@@ -108,12 +108,16 @@ Here are some resources to get you started with Earthly
 
 * 🏁 [Getting started guide](https://docs.earthly.dev/guides/basics)
 * 👀 [Examples](./examples)
-  * [Go](./examples/go)
+  * [C](./examples/c)
+  * [Cobol](./examples/cobol)
+  * [C++](./examples/cpp)
   * [Java](./examples/java)
   * [JS](./examples/js)
-  * [C++](./examples/cpp)
+  * [Go](./examples/go)
```

Comment:
can you sort these alphabetically

Reply:
Sorry for the mistake. I have rectified it.

Reply:
can you sort all of the language examples.

Reply:
Just for clarification, you want me to add all the remaining language examples and then sort them accordingly.

Reply:
I'm just talking about the current list in this README. e.g. `C`, `C++`, `Cobol`, ...

Reply:
Is this ok now?